en/advanced/gimbal_control.md

```diff
@@ -1,20 +1,37 @@
 # Gimbal Control Setup
 
-PX4 contains a generic mount/gimbal control driver with different input and output methods. Any input can be selected to drive any output.
+If you want to control a gimbal with a camera (or any other payload) attached to
+the vehicle, you need to configure how you want to control it and how PX4 can
```

Comment:
My point is that the line width of 80 is fine when editing offline and for text editor purposes, but it doesn't look good when published. I feel that a paragraph should be 1 continuous _line_, such that it flows properly in a browser.